README.md

```diff
@@ -81,3 +81,21 @@ For the information on how to run tests and configure them, go to the [`tests`](
 ## Troubleshooting
 
 > **TIP:** To solve most of the problems with Busola development, clear the browser cache or do a hard refresh of the website.
+
+## Busola in Docker against a k3d cluster
```

Comment:
```suggestion
## Symptom
You are experiencing connectivity problems with Busola in Docker against a k3d cluster.
```

We are trying to establish a fixed headline structure for troubleshooting topics :)